src/auth/authentication-provider/affinidi-authentication-provider.ts

```diff
@@ -176,11 +207,14 @@ export class AffinidiAuthenticationProvider implements AuthenticationProvider, D
   }
 
   handleExternalChangeSession = (newValue: unknown, oldValue: unknown): void => {
+    //@ts-ignore
     const oldSession = oldValue ? assertSession(oldValue) : null
+    //@ts-ignore
     const newSession = newValue ? assertSession(newValue) : null
```

Comment:
Not asking you to do this now, but if you get a chance take a look at Zod library https://github.com/colinhacks/zod. Its an awesome library that could help us do typescript safe schema validation.

Reply:
will check